packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2742,4 +2742,29 @@ export default class Meeting extends StatelessWebexPlugin {
   unlockMeeting() {
     return MeetingUtil.unlockMeeting(this.inMeetingActions, this.meetingRequest, this.locusUrl);
   }
+
+  /**
+   * Sends DTMF tones to the current meeting
+   * @param {String} tones
+   * @returns {Promise}
+   * @public
+   * @memberof Meeting
+   */
+  sendDtmf(tones) {
+    let errorMessage;
```

Comment:
A suggestion on this, not a requirement: Create a class-member-local method to generate the error instead:
```js
const reject = (message) => {
  LoggerProxy.logger.error(message);
  return Promise.reject(new Error(message);
}

if(condition) { return reject('message'); }
```
@mwegman suggested something similar to this in the `services=>serviceHost` class methods and it worked well.

Reply:
Nice solution! Implemented!